### ISSUE TITLE
Adding Cost Per Person Simulator for Organizer/Pledges

### ISSUE THREAD

**Issue by markstm17** [ArgoBus/Argo-Project-V0#184]
This relates to #135 

I think we need this for two separate reasons:
- For the organizers perspective when wanting to create a **threshold** (whether its a number of people or a cost per person threshold)
- For pledges that can see what their cost will be if they get more people to sign up

# For Argo Organizer

- Sign up/log in
- Input specific information for desired trip
 - Given range for hourly rate ($40-$50) and #167 breaks down the price-o-meter before anyone has pledged
- The CPP simulator then allows the organizer to slide from left to right (across the price-o-meter) at different amounts of people that have "pledged through the simulation" to see what the cost per person would be. It will help give an idea to the organizer how many people he or she needs to pledge. 

# For User/Pledges

- Receive link
- Sign up/log in
- Visually see:
 - What cost was
 - What it currently is 
 - What it could be
- Pledge if desired to do so
- Visually see (through simulation) how many more people are needed to reduce to certain cost per person (whether public or private bus)
 - Public bus is a range where the price decreases at 3-5 different points across different capacities 
 - Private bus decreases every time someone pledges

**Comment by markstm17** [comment]
I would say that this is also a higher priority ticket, BUT priority should be:

After #135 and #167 and high priority in #182 

AND

Before the low priority aspects of ticket #182

**Comment by markstm17** [comment]
Instead of the "Cost Per Person Simulator" you've completed the "Range" of prices it could be per person
- Example $31-$16 below



I'll let you close this @duncangraham if you think it should be closed now and isn't important. 

I'm currently keeping it open because I think there is still value to this later and in the future. Not urgent

**Comment by duncangraham** [comment]
i dont see anything else that needs to be done here– closing.
